content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md

```diff
@@ -0,0 +1,15 @@
+---
+description: ' Walk through and explanation of the LyTestTools example test. '
+title: LyTestTools Example Test Walkthrough
+linktitle: LyTestTools Example Test
+---
+
+These are instructions on how to get started with an example test using the LyTestTools (LTT) testing package.
+
+## Prerequisites
+
+* Completed [Getting Started](getting-started.md)
+* A properly configured [Open 3D Engine build](/docs/user-guide/build/)
+* Basic understanding of the [Pytest](https://pytest.org) module
+
+Make sure to complete all of the prerequisite steps above. Open the sample test file located at `Tools/LyTestTools/tests/integ/sanity_tests.py`. The provided example test contains in line comments to walk the user through each step. Following this walkthrough will instruct the user how to import the LyTestTools package, interact with Editor and game launcher classes, and structure tests.
```

Comment:
Should probably add o3de to ```"o3de/Tools/LyTestTools/tests/integ/sanity_tests.py"```

Reply:
I believe the convention is to assume the path starts with o3de?

Reply:
This is correct, paths are assumed to be relative to the O3DE clone.